ports/ncurses/portfile.cmake

```
@@ -0,0 +1,14 @@
+vcpkg_download_distfile(
+    URLS "https://invisible-mirror.net/archives/ncurses/current/ncurses-6.2-20210403.tgz"
+    FILENAME "ncurses-6.2-20210403.tgz"
+    SHA512 45d55a99cc4090d7f6ec58dd0a8f1b2839a0949f8073d73ed13c98989542dfda0ecf8fb8652ff2a952a8b7fcbbebac8a2070adb84522fc0084d6404c5bd1d6ad
+)
```

```suggestion
vcpkg_download_distfile(
    SOURCE_PATH
    URLS "https://invisible-mirror.net/archives/ncurses/current/ncurses-6.2-20210403.tgz"
    FILENAME "ncurses-6.2-20210403.tgz"
    SHA512 45d55a99cc4090d7f6ec58dd0a8f1b2839a0949f8073d73ed13c98989542dfda0ecf8fb8652ff2a952a8b7fcbbebac8a2070adb84522fc0084d6404c5bd1d6ad
)
```

Accidentally omitted the output variable.